src/servers/Cargo.toml

```diff
@@ -87,7 +87,10 @@ opensrv-mysql = { git = "https://github.com/datafuselabs/opensrv", rev = "a1fb4d
 opentelemetry-proto.workspace = true
 otel-arrow-rust.workspace = true
 parking_lot.workspace = true
-pgwire = { version = "0.30", default-features = false, features = ["server-api-ring"] }
+#pgwire = { version = "0.30", default-features = false, features = ["server-api-ring"] }
```

Comment:
```suggestion
```

Reply:
This is a temporary switch. Will switch back to release when the feature is ready.